conjur/api/client.py

```diff
@@ -90,12 +87,20 @@ def __init__(self,
                     if field_value:
                         on_disk_config[field_name] = field_value
                 loaded_config = on_disk_config
-                # pylint: disable=logging-fstring-interpolation
+                # Raise exception if the client was initialized in insecure mode
+                # but a follow-up request is run without the insecure flag
+                if ssl_verify is True and loaded_config['ca_bundle'] == '':
+                    raise OperationNotCompletedException(cause="The client was initialized without certificate verification, "
+                                        "even though the command was ran with certificate verification enabled. ",
+                                        solution="To continue communicating with the server insecurely, run the command "
+                                        "again with the --insecure flag. Otherwise, reinitialize the client.")
+
```

Comment:
not sure this is OperationNotCompletedException. more like invalidOperationException, or MissingArgumentsException